browser/src/control/jsdialog/Widget.TreeView.js

```diff
@@ -102,6 +102,15 @@ function _createRadioButton(parentContainer, treeViewData, builder, entry) {
 	return radioButton;
 }
 
+function _createColorPreviewButton(parentContainer, builder, entry) {
```

Comment:
please use some generic name (it's not only for color previews):
_createImageColumn for example